### ISSUE TITLE
Scroll frames not clipping PlayerModel widgets correctly

### ISSUE THREAD

**Issue by p3lim** [Stanzilla/WoWUIBugs#572]
![](https://user-images.githubusercontent.com/26496/89744203-01a3db80-daab-11ea-97de-b36801ba03fa.png)

As seen in this screenshot, PlayerModel widgets are not being clipped correctly while scrolling through the scroll frame, but all other widgets (e.g. the border around the model) work fine.  
From the screenshot you can see that it _sometimes_ works (the first model), but usually not.

`SetClipsChildren` also has no effect on this.

This has been an issue since atleast 9.x, but probably older than that.

**Comment by Ghostopheles** [comment]
Might be worth retesting this on 11.1.5 when able.

**Comment by p3lim** [comment]
Yeah seems to be fixed in 11.1.5, didn't even need to use `SetClipsChildren`.
